### ISSUE TITLE
Strange sorting of files in the side window.

### ISSUE THREAD

**Issue by vmacsimushkin** [inkle/ink#809]
In the side window, the files are displayed in a strange order. Before the update, they were always sorted by name. I also noticed that when editing text, the order of files changes. They are scattered differently each time.
<img width="181" alt="mission26-0 ink 2023-01-16 08-42-46" src="https://user-images.githubusercontent.com/80695782/212606608-a4996c31-d656-4dea-bc3e-878cb895e494.png">

**Comment by joningold** [comment]
in 0.14.1 the should be sorted by the order the INCLUDE statements appear in the main file, and this should update in real-time if you move those lines around. In the screenshot, however, you're listing the "unused files" -- files not linked into the main story. Inky doesn't know how to sort those, so that'd account for the behaviour. 

To fix this, we'll need to specifically sort the unused file list differently than the other file lists, and fall back to an alphabetical sort in this case.

**Comment by vmacsimushkin** [comment]
> To fix this, we'll need to specifically sort the unused file list differently than the other file lists, and fall back to an alphabetical sort in this case.

Yes, you're right. This is a list of files that just lie in one folder. It's so convenient for me to create a lot of small files, which are then combined in the game. In previous versions, they were all sorted alphabetically :)

Well, I got the answer. I hope it will be possible to arrange sorting.
